nad/epsg

```diff
@@ -1,3 +1,4 @@
+<metadata> +version=9.0.0 +origin=EPSG +lastupdate=2017-01-10
```

Comment:
It is probably easiest to just to do it manually, although it would be cool to have the script to it automatically.

What do you think about this way of providing the version number of the EPSG database?